GenerateTestPfx.ps1

```diff
@@ -0,0 +1,12 @@
+$CertificateFriendlyName = "WinUITest"
+$Publisher = "CN=WinUITest"
+
+$cert = New-SelfSignedCertificate -Type Custom `
+    -Subject $Publisher `
+    -KeyUsage DigitalSignature `
+    -FriendlyName $CertificateFriendlyName `
+    -CertStoreLocation "Cert:\CurrentUser\My" `
+    -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.3", "2.5.29.19={text}")
```

Comment:
Does this mean anything, or is this just a magic incantation that works?

Reply:
It's pretty opaque alright. Here is where I got it from:
https://docs.microsoft.com/en-us/windows/msix/package/create-certificate-package-signing#create-a-self-signed-certificate